setup.cfg

```
@@ -45,7 +45,7 @@ docs =
     matplotlib
 
 [options.package_data]
-skypy = data/*,*/tests/data/*
+skypy = data/*,data/spectrum_templates/*,*/tests/data/*
```

Would prefer this for if we ever add other data.

```suggestion
skypy = data/*,data/*/*,*/tests/data/*
```